src/main/resources/catalog.json

```diff
@@ -0,0 +1,181 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the license for the specific language governing permissions and
+ * limitations under the license.
+ */
+ {
+  "products" : [ ],
+  "categories" : [ ],
+  "events" : [ {
+    "id" : 1,
+    "name" : "login",
+    "displayName" : "Login",
+    "description" : "User Login",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    } ]
+  }, {
+    "id" : 2,
+    "name" : "updateKey",
+    "displayName" : "Update Key",
+    "description" : "User updates API key(s)",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "requestUri",
+      "required" : false
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    } ]
+  }, {
+    "id" : 3,
+    "name" : "updatePassword",
+    "displayName" : "Update Password",
+    "description" : "User updates password",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "requestUri",
+      "required" : false
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    } ]
+  }, {
+    "id" : 4,
+    "name" : "logout",
+    "displayName" : "Logout",
+    "description" : "User Logout",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    } ]
+  }, {
+    "id" : 5,
+    "name" : "createUser",
+    "displayName" : "Create User",
+    "description" : "Create User",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
+    }, {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "requestUri",
+      "required" : false
+    } ]
+  } ],
+  "attributes" : [ {
+    "id" : 3,
+    "name" : "userId",
+    "displayName" : "UserId",
+    "description" : "Id of the User",
+    "dataType" : "INT",
+    "indexed" : true,
+    "sortable" : true,
+    "required" : true,
+    "requestContext" : true,
+    "examples" : null,
+    "aliases" : [ ],
+    "constraints" : [ ],
+    "catalogId" : "DEFAULT"
+  }, {
+    "id" : 11,
+    "name" : "requestId",
+    "displayName" : "RequestId",
+    "description" : "Used as a correlation id for logs",
+    "dataType" : "STRING",
+    "indexed" : true,
+    "sortable" : true,
```

Comment:
I don't think this will end up being sortable. This field will usually be something like a UUID which is random.